static/src/js/utils/removeMetadataKeys.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Given a ummMetadata object, removes all the keys that are given in the
+ * keys array.
+ * @param {Object} metadata An object containing the keys and value
+ * @param {Array} keys An array that has all the keys that needs to be removed
+ */
+const removeMetadataKeys = (metadata, keys) => {
+  keys.forEach((key) => {
```

Comment:
Rather than modify the existing object, it would probably be safer to return a new object without the keys. Things can get funky when modifying objects that are passed around by reference. This is why it triggers the no-param-reassign warning.